<launch>
    <node pkg="human_tracking" name="user_control" type="user_control.py" output="screen">
        <param name="color_topic" value="/camera/color/image_raw/compressed"/>
        <param name="depth_topic" value="/camera/aligned_depth_to_color/image_raw"/>
        <param name="depth_camera_info" value="/camera/aligned_depth_to_color/camera_info"/>
    </node>
</launch>
